docs/hop-tech-manual/modules/ROOT/pages/docker-container.adoc

```diff
@@ -270,7 +270,7 @@ We also make it parameter-driven, so it any other team can use it.We create our
 
 [source,dockerfile]
 ----
-FROM apache/incubator-hop:0.70-SNAPSHOT
+FROM apache/incubator-hop:1.1.0-SNAPSHOT
```

Comment:
This should probably become 

`FROM apache/hop:Development`

or 

`FROM apache/hop:latest`

Reply:
fair point. I didn't make any changes to any of the repository or other incubator URLs, as they would all be incorrect at this point. 
We'll need to make another pass through the docs and website once the infrastructure changes have been made imho.